src/components/managedVaults/community/vaultDetails/VaultPosition.tsx

```diff
@@ -0,0 +1,70 @@
+import DisplayCurrency from 'components/common/DisplayCurrency'
+import PositionInfo from './common/PositionInfo'
+import Text from 'components/common/Text'
+import useStore from 'store'
+import { ArrowDownLine } from 'components/common/Icons'
+import { BN } from 'utils/helpers'
+import { BNCoin } from 'types/classes/BNCoin'
+import { FormattedNumber } from 'components/common/FormattedNumber'
+import { PRICE_ORACLE_DECIMALS } from 'constants/query'
+import { useManagedVaultUserDeposits } from 'hooks/managedVaults/useManagedVaultUserDeposits'
+
+interface Props {
+  totalVaultTokens: string
+  baseDenom: string
+  tokenDenom: string
+  isOwner: boolean
+  onDeposit: () => void
+  onWithdraw: () => void
+}
+
+export default function VaultPosition(props: Props) {
+  const { totalVaultTokens, baseDenom, isOwner, tokenDenom, onDeposit, onWithdraw } = props
+
+  const address = useStore((s) => s.address)
+  const { amount, calculateVaultShare } = useManagedVaultUserDeposits(address, tokenDenom)
+  const sharePercentage = calculateVaultShare(totalVaultTokens)
+  const noDeposits = amount === '0'
+
+  return (
+    <PositionInfo
+      value={
+        !noDeposits ? (
+          <DisplayCurrency
+            coin={BNCoin.fromDenomAndBigNumber(
+              baseDenom,
+              BN(amount).shiftedBy(-PRICE_ORACLE_DECIMALS),
+            )}
+            className='text-2xl'
+          />
+        ) : (
+          <Text>No deposits</Text>
+        )
```

Comment:
Do not invert an already inverted statement.
 
```suggestion
        noDeposits ? (
          <Text>No deposits</Text>
        ) : (
          <DisplayCurrency
            coin={BNCoin.fromDenomAndBigNumber(
              baseDenom,
              BN(amount).shiftedBy(-PRICE_ORACLE_DECIMALS),
            )}
            className='text-2xl'
          />
        )
```